npm/webpack-batteries-included-preprocessor/package.json

```diff
@@ -32,12 +32,12 @@
     "json5": "2.2.3",
     "os-browserify": "^0.3.0",
     "path-browserify": "^1.0.1",
-    "pnp-webpack-plugin": "^1.7.0",
     "process": "^0.11.10",
     "punycode": "^2.3.0",
     "querystring-es3": "^0.2.1",
     "stream-browserify": "^3.0.0",
     "stream-http": "^3.2.0",
+    "string_decoder": "1.3.0",
```

Comment:
we actually missed `string_decoder` on the first pass of updating this from webpack 4 to 5. there likely werent issued filed since `string_decoder` ships with the cypress binary. This would only be obvious in a handlful of scenarios, like using yarn 4 PnP and installing the WBIP preprocessor standalone and configuring it.